Update teams-store-validation-guidelines.md (#4138)
MicrosoftDocs/msteams-docs: msteams-platform/concepts/deploy-and-publish/appsource/prepare/teams-store-validation-guidelines.md

* Tabs in a personal context may aggregate content from shared instances of the app.
* Tabs must be responsive to Teams themes. When a user changes the theme, the app's theme must reflect the selection.
* Tabs must use Teams-styled components, such as, Teams fonts, type ramps, color palettes, grid system, motion, tone of voice, and so on whenever possible.

> * Include a 'Settings' tabs if your app has functionality that requires a changes in the settings.

Reviewer: Please rephrase:
```suggestion
* If your app functionality requires changes in the settings, include a **Settings** tab.
```
